taccsite_cms/settings.py

```diff
@@ -241,6 +241,11 @@ def getsecrets():
     # TODO: Extract TACC CMS UI components into pip-installable plugins
     # FAQ: The djangocms_bootstrap4 library can serve as an example
     'taccsite_cms.contrib.taccsite_sample',
+    'taccsite_cms.contrib.taccsite_static_article_list',
+    'taccsite_cms.contrib.taccsite_static_article_preview',
+    'taccsite_cms.contrib.taccsite_blockquote',
+    'taccsite_cms.contrib.taccsite_offset',
```

Comment:
These cause an error on initialization (I had to comment them to make it run properly).

Reply:
That is a problem. Three of those should not be there. This problem suggests that more code may be present that should not be. I will investigate. Thank you.

<details>
<summary>Details</summary>

_There are so many dependent branches [for Frontera System Specs page](https://github.com/TACC/Core-CMS/pull/313) and [Frontera Homepage plugins](https://github.com/TACC/Core-CMS/pull/259) that I am trying to author independently but test together 🤕 and I crossed the streams._

</details>

Reply:
I have removed those lines (via 5ed2892). No related erroneous code was found, but I found and resolved\* a different unexpected change in the submodule diff.

<details>
<summary>Footnotes</summary>

\* I resolved unexpected submod diff with a merge of `main`, a2807a2 (after a submod pointer update: #315).

</details>